src/js/common/schemaform/widgets/PhoneNumberWidget.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TextWidget from './TextWidget';
+
+/*
+ * Handles removing dashes from SSNs by keeping the user input in local state
+ * and saving the transformed version instead
+ */
+export default class PhoneNumberWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { val: props.value };
+  }
+  handleChange = (val) => {
+    const stripped = val.replace(/[\D]/g, '');
```

Comment:
This line (14) is the only difference aside from naming between this component and `SSNWidget`. Perhaps we could consider making a `StrippedText` widget or something that takes a string `value` and a replacement regex?

Again, I'm not sure how to use it in the schema, is the problem.